includes/Media/Types.php

```diff
@@ -82,16 +61,22 @@ public function get_file_type_exts( array $mime_types = [] ): array {
 	 */
 	public function get_allowed_mime_types(): array {
 		$default_allowed_mime_types = [
-			'image' => [
+			'image'  => [
 				'image/webp',
 				'image/png',
 				'image/jpeg',
 				'image/jpg',
 				'image/gif',
 			],
-			// TODO: Update once audio elements are supported.
-			'audio' => [],
-			'video' => [
+			'audio'  => [
+				'audio/mpeg',
+				'audio/aac',
+				'audio/wav',
+				'audio/ogg',
+			],
+			'text'   => [ 'text/vtt' ],
```

Comment:
What's the reason for adding this?

`text/vtt` should not be passed via configuration, that one is never going to change. It's why it's not passed via config today.

Reply:
So it runs through the mime type filtering to ensure that this file type can be uploaded. At the moment, these is no check for this and we just guess that vvt files can be uploaded.